lib/awful/client/shape.lua

```diff
@@ -51,8 +51,54 @@ function shape.get_transformed(c, shape_name)
     return result
 end
 
+--- Set the client shape.
+-- @function awful.client.shape.set
+-- @client c The client whose shape should be retrieved
+-- @tparam gears.shape A gears.shape compatible function.
+-- @see gears.shape
+function shape.set(c, shape)
```

Comment:
`function client.object.set_shape` is still required to eixst in `client.lua`. Otherwise `c.shape = ...` wont work